packages/compass-workspaces/src/types.ts

```diff
@@ -16,6 +16,8 @@ export type MyQueriesWorkspace = {
 export type ShellWorkspace = {
   type: 'Shell';
   connectionId: string;
+  initialEvaluate?: string | string[];
+  initialInput?: string;
```

Comment:
My first idea is to allow to pass those props directy to the compass-shell, this gives us a lot of flexibility when opening shell, but means that this initial state needs to be formed in place when open method is called. Another approach we can take is to make those more specific and make shell the place that figures out what is the initial state that needs to be applied. Don't have a strong preference here which direction to take, so open to suggestions and happy to change this before merging.